Makefile

```diff
@@ -755,6 +755,12 @@ test-k8se2e-only: ## Run k8s network conformance test, use TYPE=basic for only d
 	cd hack/scripts && bash ./k8se2e-tests.sh $(OS) $(TYPE)
 	cd ../..
 
+# install azure Linux CNS and CNI dropgz images
+install-azure-images:
```

Comment:
If we use `test-integration`, what is the issue in that case ? `make test-integration` starts with setup_test.go.
If we just want to install the binaries, I think we should extract that part of the code as a separate function to provide the same functionality.

Reply:
similar feedback to https://github.com/Azure/azure-container-networking/pull/2032#discussion_r1270926850